content/trial_getting_started/set_trial_manager/download_community.md

```diff
@@ -36,5 +36,7 @@ ____
 
 What's next?
 
-* Optional - When using a remote Docker service, working with Cloudify using a local CLI is the recommended approach. [Install local Cloudify CLI]({{< relref "/install_maintain/installation/installing-cli.md" >}}).
-* Go ahead and examine our [example based tutorials]({{< relref "trial_getting_started/examples/_index.md" >}}) to learn about Cloudify basics.
+* To test your installation using the command line utility on your docker image refer to the [local CLI guide] ({{< relref "trial_getting_started/set_trial_manager/getting-started-with-cloudify-docker-and-cli" >}})
```

Comment:
Right - I added the CLI installation to the local CLI guide. I also made sure that the CLI command reference would also point to the CLI installation.
Since in this case the user is already using docker starting with the local CLI guide makes more sense as it doesn't require any additional installation.